base_geoengine/fields.py

```diff
@@ -56,9 +57,22 @@ def convert_to_column(self, value, record):
         else:
             return shape_to_write.wkt
 
+    def convert_to_cache(self, value, record, validate=True):
+        val = value
+        if isinstance(value, BaseGeometry):
+            val = value.wkb_hex
+        return val
+
+    def convert_to_record(self, value, record):
+        if value:
+            return self.load_geo(value)
```

Comment:
I changed my code. I also discovered a new case which happen during onchange methods. When a onchange computed a GeoPoint, given value in convert_to_record method corresponds to the value returned by the convert_to_read method.